npm/asciidoctor-revealjs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+'use strict'
+
+process.title = 'asciidoctor-revealjs'
+
+const { Options, Invoker } = require('@asciidoctor/cli')
+const asciidoctorRevealjs = require('@asciidoctor/reveal.js')
+asciidoctorRevealjs.register()
+
+const options = new Options({backend: 'revealjs'}).parse(process.argv)
+
+class RevealJsInvoker extends Invoker {
+  printVersion () {
```

Comment:
@obilodeau You can use `@asciidoctor/cli` version 3.1.0 and override the `version` function:

```js
version () {
  const pkg = require('../package.json')
  return `Asciidoctor reveal.js ${pkg.version} using ${super.version()}`
}
```